### PR TITLE
doc: Update README

### PR DESCRIPTION
**Pull request by Onkar627**
(_edit by @skaldarnar_)

This PR improves the language of the player section in the README.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -67,18 +67,18 @@ We encourage contributions from anybody and try to keep a warm and friendly comm
...
 For easy game setup (recommended) you can use our launcher - [download it here.](https://github.com/MovingBlocks/TerasologyLauncher/releases)
 
 For more information about playing, like hot keys or server hosting, see the [dedicated page](docs/Playing.md) or check out the [modules.](docs/Modules.md)
 
 
-#### Alternate Install Method
+#### Alternative Installation Methods
 
-If you know you already have a Java Runtime Environment installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
+If you know you already have a Java Runtime Environment(JRE) installed, you may use a direct download release as an alternative for using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
```
Comment (skaldarnar) [inline README.markdown:79]: I do think that "alternative to ..." was correct here, as the launcher is one option, and we're pointing out another option. See https://english.stackexchange.com/questions/3835/alternative-to-vs-alternative-for

:monocle_face: <-- that's how I feel every time I look something up on _English Language StackExchange_

File: README.markdown [hunk 2]
```diff
@@ -67,18 +67,18 @@ We encourage contributions from anybody and try to keep a warm and friendly comm
...
 
-Direct download stable builds are uploaded to [our release section here on GitHub](https://github.com/MovingBlocks/Terasology/releases) while the cutting-edge develop version can be downloaded direct [here from our Jenkins](http://jenkins.terasology.org/job/DistroOmega/lastSuccessfulBuild/artifact/distros/omega/build/distributions/TerasologyOmega.zip)
+Direct download stable builds are uploaded to [our release section here on GitHub](https://github.com/MovingBlocks/Terasology/releases) while the cutting-edge developed version can be downloaded direct [here from our Jenkins](http://jenkins.terasology.org/job/DistroOmega/lastSuccessfulBuild/artifact/distros/omega/build/distributions/TerasologyOmega.zip)
```
Comment (skaldarnar) [inline README.markdown:81]: This one is a bit different. With "develop version" we try to refer to the `develop` branch. Maybe "development version" is a somewhat common term used for these kinds of builds?

**Review by skaldarnar — CHANGES_REQUESTED** [review]
Thanks for the PR, highly appreciated!

I've left two comments, the first to revert that change, and the second as suggestion to make the wording a bit clearer =) 

Please let us know if you have any trouble trying to incorporate those changes :upside_down_face:

File: README.markdown [hunk 3]
```diff
@@ -35,7 +35,7 @@ We encourage contributions from anybody and try to keep a warm and friendly comm
     </tr>
     <tr>
         <td>System (OS)</td>
-        <td colspan="2" align="center">Windows, MacOS, Linux (64 bit)</td>
+        <td colspan="2" align="center"> Windows, MacOS, Linux (64 bit)</td>
```
Comment (jdrueckert) [inline README.markdown:38]: Assuming, this additional whitespace got in by accident?
```suggestion
        <td colspan="2" align="center">Windows, MacOS, Linux (64 bit)</td>
```

File: README.markdown (hunk 1 above)
Comment (jdrueckert) [inline README.markdown:79]: Taking that whitespace from above and adding it here :wink: so overall the whitespace count was good :grin: :+1: 
```suggestion
If you know you already have a Java Runtime Environment (JRE) installed, you may use a direct download release as an alternative for using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
```

Comment (Onkar627) [reply README.markdown:79]: > Thanks for the PR, highly appreciated!
> 
> I've left two comments, the first to revert that change, and the second as suggestion to make the wording a bit clearer =)
> 
> Please let us know if you have any trouble trying to incorporate those changes 🙃

Thank you very much for reviewing! Will incorporate the changes

File: README.markdown [hunk 4]
```diff
@@ -67,16 +67,16 @@ We encourage contributions from anybody and try to keep a warm and friendly comm
...
 For easy game setup (recommended) you can use our launcher - [download it here.](https://github.com/MovingBlocks/TerasologyLauncher/releases)
 
 For more information about playing, like hot keys or server hosting, see the [dedicated page](docs/Playing.md) or check out the [modules.](docs/Modules.md)
 
 
-#### Alternate Install Method
+#### Alternative Installation Methods
 
-If you know you already have a Java Runtime Environment installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
+If you already have Java Runtime Environment(JRE) installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
```
Comment (skaldarnar) [inline README.markdown:79]:
```suggestion
If you already have Java Runtime Environment (JRE) installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
```

**Review by skaldarnar — APPROVED** [review]

**Pull request merged**